### PR TITLE
Source COde

### PR DESCRIPTION
**Pull request by ranjithrk007**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (philipnavin) [inline .idea/vcs.xml:1]: should be git ignored

File: .idea/workspace.xml [hunk 2]
```diff
@@ -1,12 +1,114 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (philipnavin) [inline .idea/workspace.xml:1]: should be git ignored

**Review by philipnavin — CHANGES_REQUESTED** [review]

File: src/leapyear.txt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+        int year = s1.nextInt();
+        isLeapYear(year);
+    }
+
+    public static void isLeapYear(int year){
+        if (year<=1 && year >= 9999){
+            System.out.println("false");
+
+        }else if ((year % 4 == 0) && (year % 100 == 0) && (year % 400 == 0)){
```
Comment (philipnavin) [inline src/leapyear.txt:19]: Why do we want to check `(year % 4 == 0)` and `(year % 400 == 0)`

**Review by philipnavin — CHANGES_REQUESTED** [review]

File: .idea/vcs.xml (hunk 1 above)
Comment (ranjithrk007) [reply .idea/vcs.xml:1]: added vcs.xml in .gitignore file

File: .idea/workspace.xml (hunk 2 above)
Comment (ranjithrk007) [reply .idea/workspace.xml:1]: added workspace.xml in .gitignore file

**Pull request closed without merging**